mangaki/mangaki/api/cards.py

```diff
@@ -0,0 +1,119 @@
+import enum
+
+from rest_framework.decorators import api_view, permission_classes, throttle_classes
+from rest_framework import serializers
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.throttling import UserRateThrottle
+
+from mangaki.models import Category, Work
+from mangaki.utils.ratings import current_user_ratings
+
+NB_POINTS_DPP = 10
+POSTERS_PER_PAGE = 24
+
+
+class SlotCardTypes(enum.Enum):
+    popularity = 'popular'
+    controversy = 'controversial'
```

Comment:
At the start, I wanted to have something like: `queryset[SlotCardTypes.popularity]()`

But, due to DPP and random special treatment, that does not sound possible.

As a relic of the past, I kept the mapping. Shouldn't we uniformize them to match the queryset methods?